server_environment_ir_config_parameter/models/ir_config_parameter.py

```diff
@@ -51,4 +51,4 @@ def write(self, vals):
                 newvals = dict(vals, value=serv_config.get(SECTION, key))
             else:
                 newvals = vals
-            super(IrConfigParameter, rec).write(newvals)
+        return super(IrConfigParameter, rec).write(newvals)
```

Comment:
@rousseldenis this does not look right. The write must be inside the loop as we overwrite whatever is provided in vals with the values found in the config file, so each property may end up with a different value. So this commit changes the expected behaviour.